src/L.Control.Heightgraph.js

```diff
@@ -461,25 +484,25 @@ L.Control.Heightgraph = L.Control.extend({
         this._focusDistance = this._focus.append("text")
             .attr("x", 7)
             .attr("y", -this._y(boxPosition) + textDistance)
-            .attr("id", "distance")
+            .attr("id", "heightgraph.distance")
```

Comment:
In embed mode, there could be other elements on the page with these ids `distance`, `height`, etc. I gave them more specific ids to avoid conflicts.

Reply:
Excellent! Related to #65